src/table/context/TableContext.tsx

```diff
@@ -3,9 +3,9 @@ import { stardust } from '@nebula.js/stardust';
 import PropTypes from 'prop-types';
 import { createSelectorProvider } from './createSelectorProvider';
 import { reducer } from '../utils/selections-utils';
-import { ExtendedSelectionAPI } from '../../types';
+import { ExtendedSelectionAPI, AppContextInterface } from '../../types';
 
-export const TableContext = createContext({});
+export const TableContext = createContext<AppContextInterface | null>(null);
```

Comment:
have you seen my approach to this? Not sure which one is better. https://github.com/qlik-oss/sn-table/pull/646

Reply:
Not yet, let me see.